prusti-encoder/src/encoders/mir_impure.rs

```diff
@@ -628,7 +584,7 @@ impl<'tcx, 'vir, 'enc> mir::visit::Visitor<'tcx> for EncoderVisitor<'tcx, 'vir,
                 // multiple assignments performed in multiple statements. In
                 // such a case, `expr` is left as `None`.
```

Comment:
This comment (+ lines above which I cannot select) is stale with the changed `Aggregate` case. Update it to match the behaviour.